<launch>

  <node pkg="mpu6050_serial_to_imu" type="mpu6050_serial_to_imu_node" name="mpu6050_serial_to_imu_node" output="screen">
    <param name="port" value="/dev/vk_imu"/>
    <param name="baudrate" value="115200"/>
    <param name="frame_id" value="imu_link"/>
 	<param name="linear_acceleration_stddev" value="0.04"/>
 	<param name="angular_velocity_stddev" value="0.02"/>
 	<param name="orientation_stddev" value="0.0025"/>
  </node>

  <!--node pkg="rviz" type="rviz" name="rviz" args="-d $(find mpu6050_serial_to_imu)/rviz/demo.rviz" required="true"/-->

</launch>
